### PR TITLE
Added a documentation regarding support

### PR DESCRIPTION
**Pull request by MikeRys**
Added a documentation regarding support

### CODE REVIEW

**Review by Niharikadutta — DISMISSED** [review]
LGTM, thanks @MikeRys

File: README.md [hunk 1]
```diff
@@ -167,6 +167,11 @@ This project would not have been possible without the outstanding work from the
 
 The .NET for Apache Spark team encourages [contributions](docs/contributing.md), both issues and PRs. The first step is finding an [existing issue](https://github.com/dotnet/spark/issues) you want to contribute to or if you cannot find any, [open an issue](https://github.com/dotnet/spark/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+).
 
+## Support
```
Comment (suhsteve) [inline README.md:170]: Can you update the table of contents to link to this

Comment (suhsteve) [reply README.md:171]: nm, I went ahead and edited the PR.

**Review by suhsteve — APPROVED** [review]
LGTM, thanks @MikeRys

**Pull request merged**